romana-install/group_vars/aws

```diff
@@ -1,6 +1,12 @@
-key_name: "shared-key-1"
-
+ec2_keypair: ""
 region: "us-west-1"
-instance_type: "t2.large"
+instance_type:
+  controller:
+    devstack: "t2.large"
+    kubernetes: "t2.small"
+  compute:
+    devstack: "t2.small"
+    kubernetes: "t2.small"
+
 controller_image: "ami-bf3ec1fb" # Ubuntu 14.04.3 LTS
```

Comment:
That's fine, but I would suggest that we don't advertise the region as a configurable parameter then.